File: 프로그래머스/1/12926. 시저 암호/시저 암호.java
import java.util.*;
class Solution {
    public String solution(String s, int n) {
        String answer = "";
        for (int i = 0; i < s.length(); i++) {
            char c = s.charAt(i);
            if (c == ' ') {
                answer += " ";
                continue;
            }
            if (c >= 'a' && c <= 'z') {
                char a = 'a';
                int n2 = 97;
                if (c + n > 'z') {
                    answer += Character.toString((char)(c + n - 26));
                } else {
                    answer += Character.toString(c + n);
                }
            } else if (c >= 'A' && c <= 'Z') {
                char A = 'A';
                if (c + n > 'Z') {
                    answer += Character.toString((char)(c + n - 26));
                } else {
                    answer += Character.toString(c + n);
                }
            }
            
        }
        return answer;
    }
}

File: 프로그래머스/1/12926. 시저 암호/시저 암호.py
def solution(s, n):
    
    
    answer = ''
    for i in s:
        if i == " ":
            answer = answer + " "
            continue;
#        print(i)
#        print(ord(i))
        if ord(i) >= 65 and ord(i) <= 90:
            print(ord(i)+n)
            print((ord(i)+n)%90)
            answer = answer + chr((ord(i)+n-65)%26+65)
        else: # 97 122
            print(ord(i)+n)
            print((ord(i)+n)%122)
            answer = answer + chr((ord(i)+n-97)%26+97)
    return answer
    
